README.md

```diff
@@ -95,18 +95,29 @@ Latest installers with Mamba and PyPy in the base environment:
 
 ## Install
 
-### Unix-like platforms (Mac OS & Linux)
+### Unix-like platforms (macOS & Linux)
 
 Download the installer using curl or wget or your favorite program and run the script.
 For eg:
 
-    curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
-    bash Miniforge3-$(uname)-$(uname -m).sh
+```bash
+curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
+bash Miniforge3-$(uname)-$(uname -m).sh
+```
 
 or
 
-    wget "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
-    bash Miniforge3-$(uname)-$(uname -m).sh
+```bash
+wget "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
+bash Miniforge3-$(uname)-$(uname -m).sh
+```
+
+**NOTE:** For macOS, you may need to run `bash` with `sudo`:
```

Comment:
I'm not sure this is correct.

We actually prefer that users not use sudo unless they really know what they are doing, then they don't need us to tell them to use sudo. The preferred way is simply to install in your user's home directory like the default will do.

Can you expand on the usecase where you find `sudo` necessary or desired?

Reply:
it should be attempting to change your personal zshrc which you should have access to.

That is, it should be trying to edit (on OSX)
```
/Users/fakerybakery/.zshrc
```

which should be a file you control, that is writable by your user.

It maybe that you are running into a real bug.

If you can recreate the traceback and the error that would be really useful.

I don't have an OSX machine, but others might be able to help troubleshoot.

Reply:
I got:
```
TypeError: memoryview: a bytes-like object is required, not 'str'
```

Reply:
This is likely a bug in the installer code. If you can share more of the traceback (or better yet how the install was done), that would help in identifying where this comes from